master/contrib/Dockerfile

```diff
@@ -64,13 +64,13 @@ from ubuntu:12.04
 # Install buildbot and its dependencies
 
 run /bin/echo -e "\
-    deb http://archive.ubuntu.com/ubuntu precise main universe\n\
-    deb http://archive.ubuntu.com/ubuntu precise-updates main universe" > \
+    deb http://archive.ubuntu.com/ubuntu trusty main universe\n\
+    deb http://archive.ubuntu.com/ubuntu trusty-updates main universe" > \
     /etc/apt/sources.list
 run apt-get update
 run DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip python-dev \
     supervisor git sudo ssh
-run pip install sqlalchemy==0.7.9 buildbot buildbot_slave
+run pip install -v sqlalchemy==0.7.9 buildbot buildbot_slave
```

Comment:
When I am trying with the original docker file, it stops here a lot of time outputing nothing, so I think verbose with level one is better?

BTW: the `pip install` instructions in the traditional [firstrun](http://docs.buildbot.net/current/tutorial/firstrun.html) page installs `sqlalchemy==0.7.9`. Maybe we should updated here too?